api/src/main/java/de/adorsys/keymanagement/api/generator/EncryptingKeyGenerator.java

```diff
@@ -3,7 +3,15 @@
 import de.adorsys.keymanagement.api.types.entity.KeyPairEntry;
 import de.adorsys.keymanagement.api.types.template.generated.Encrypting;
 
+/**
+ * Specific interface for generating encryption keys
+ */
 public interface EncryptingKeyGenerator {
 
+    /**
+     * generates key pair using template
+     * @param fromTemplate encrypting template which is used to generate key pair
```

Comment:
It is better to expand `encrypting template which is used to generate key pair` to `Template that defines encrypting key properties like encryption algorithm, key size, key name that are going to be used on creation`

Reply:
done